content/en/shift_left.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Shift Left
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Shift Left is the practice of implementing tests, security, or other development practices early in the software development cycle rather than towards the end. The actual stages in the software development lifecycle may be defined in different ways based on the team and the models used. When considered as a left-to-right workflow, early stage activities such as design, planning, and initial development fall on the left side of the lifecycle, and the workflow continues moving right into stages such as testing, deployment, and operations.
```

Comment:
```suggestion
Left in Shift Left refers to the left-to-right (development-to-development) workflow. Shift Left is the practice of implementing tests, security, or other development practices early in the software development cycle rather than towards the end. The actual stages in the software development lifecycle may be defined in different ways based on the team and the models used.
```

Reply:
I think the explanation should go first. Otherwise, they'll have to re-read the first paragraph once they find out what left-to-right is. I moved it up which ended up looking like below. However, it's a LOT of explanation before getting to the actual definition. Maybe one sentence as suggested above is enough?

> Left in Shift Left refers to the left-to-right (development-to-development) workflow. Early-stage activities such as design, planning, and initial development fall on the left side of the lifecycle, whereas the workflow continues moving right into stages such as testing, deployment, and operations.

Shift Left is the practice of implementing tests, security, or other development practices early in the software development cycle rather than towards the end. The actual stages in the software development lifecycle may be defined in different ways based on the team and the models used.